devito/ir/clusters/cluster.py

```diff
@@ -188,8 +188,8 @@ def dtype(self):
 
     @cached_property
     def ops(self):
-        """The Cluster operation count."""
-        return self.ispace.size*sum(estimate_cost(i) for i in self.exprs)
+        """Number of operations performed at each iteration."""
```

Comment:
at each grid point? I find iteration slightly missleading.

Reply:
an iteration is a lower level concept than grid, and not all clusters (or in fact, not all devito codes) may be defined on a grid, so iteration is technically more correct